pkgs/misc/lguf-brightness/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, cmake, libusb1, ncurses5 }:
+
+stdenv.mkDerivation rec {
+  pname = "lguf-brightness";
+
+  version = "unstable-2018-02-07";
+
+  src = fetchFromGitHub  {
+    owner = "periklis";
+    repo = pname;
+    rev = "d194272b7a0374b27f036cbc1a9be7f231d40cbb";
+    sha256 = "0zj81bqchms9m7rik1jxp6zylh9dxqzr7krlj9947v0phr4qgah4";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ libusb1 ncurses5 ];
+
+  installPhase = ''
+    install -D lguf_brightness $out/bin/lguf_brightness
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Adjust brightness for LG UltraFine 4K display (cross platform)";
+    homepage = https://github.com/periklis/lguf-brightness;
+    licence = licences.lgp2_1;
```

Comment:
This should be:
```
license = licenses.lgpl21Plus;
```

Both spelling of license, licenses and the name of the actual license :-)

That's why the eval breaks.

Reply:
Good catch! Sorry i was blind by nix-build working due to a previous successful build. In addition i didn't notice this mistake cause of i didn't get any notification from the checks below.